### PR TITLE
[MI-760] Display setup notiication

### PR DESCRIPTION
**Pull request by joseconsador**
/cc @zendesk/mintegrations

### Description

screen shot 2016-08-16 at 10 08 02 am

Display setup notification if the API request fails on the first call.
css fixes, enable handlebars partials

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-760

### Risks
* Medium: Order data may not be fetched

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ dist/assets/main.css.map
 dist/assets/main.js
 dist/assets/main.js.map
 dist/translations/
+settings.yml
```
Comment (miogalang) [inline .gitignore]: this is ignored but included below?

File: src/templates/setup.hdbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="u-ta-center">
+	<div class="u-mb-sm u-light">
```
Comment (samgavinio) [inline src/templates/setup.hdbs]: Seems to be some tabbing issues here.

**Comment by joseconsador** [comment]
@miogalang @samgavinio 💇 thanks

**Comment by samgavinio** [comment]
👍

**Comment by jmramos02** [comment]
👍

**Pull request merged**